packages/rulesets/src/spectral/test/parameters-schema-as-type-object.test.ts

```diff
@@ -0,0 +1,106 @@
+import { Spectral } from "@stoplight/spectral-core"
+import linterForRule from "./utils"
+
+let linter: Spectral
+
+const ERROR_MESSAGE = "The schema for body parameters must specify type:object and include a definition for its reference model."
+
+beforeAll(async () => {
+  linter = await linterForRule("ParametersSchemaAsTypeObject")
+  return linter
+})
+
+test("ParametersSchemaAsTypeObject should find errors", () => {
+  const myOpenApiDocument = {
+    swagger: "2.0",
+    parameters: {
```

Comment:
`parameters` should be under `paths.pathName.opName`. e.g.
```json
{
  "paths": {
    "/my/example/path": {
      "get": {
        "parameters": {
          ...
        }
      }
    }
  }
}
```
https://swagger.io/docs/specification/describing-parameters/

Reply:
Updated